assets/javascripts/discourse-assign/initializers/extend-for-assigns.js.es6

```diff
@@ -319,6 +320,22 @@ export default {
       return;
     }
 
+    SearchAdvancedOptions.reopen({
+      didInsertElement() {
```

Comment:
These arrays of options are set up in `init()`. So I think it might be cleaner to override `init()` rather than `didInsertElement`